<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PersonalChatDlg</class>
 <widget class="QDialog" name="PersonalChatDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="avatar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>55</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string>avatar</string>
   </property>
  </widget>
  <widget class="QLabel" name="btn_min">
   <property name="geometry">
    <rect>
     <x>509</x>
     <y>5</y>
     <width>26</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="btn_close">
   <property name="geometry">
    <rect>
     <x>535</x>
     <y>5</y>
     <width>26</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="friend_name">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>400</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>friendname</string>
   </property>
  </widget>
  <widget class="GifTextEdit" name="announcement">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>86</y>
     <width>412</width>
     <height>218</height>
    </rect>
   </property>
  </widget>
  <widget class="GifTextEdit" name="messageEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>330</y>
     <width>412</width>
     <height>138</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'åææ¥·ä½'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Sans Serif'; font-size:9pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="acceptRichText">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>467</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#0000ff;&quot;&gt;Ctrl+Q&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/button.png);</string>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendMessageButton">
   <property name="geometry">
    <rect>
     <x>321</x>
     <y>467</y>
     <width>90</width>
     <height>25</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; color:#0000ff;&quot;&gt;Ctrl+Enter&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/button.png);</string>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Return</string>
   </property>
  </widget>
  <widget class="QLabel" name="fileTip">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>302</y>
     <width>470</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="saveFileButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
  </widget>
  <widget class="QPushButton" name="refuseButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>260</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>拒绝</string>
   </property>
  </widget>
  <widget class="QPushButton" name="sendfileButton">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>285</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>发送文件</string>
   </property>
  </widget>
  <widget class="QLabel" name="ownAvatar">
   <property name="geometry">
    <rect>
     <x>412</x>
     <y>330</y>
     <width>158</width>
     <height>163</height>
    </rect>
   </property>
   <property name="text">
    <string>ownAvatar</string>
   </property>
  </widget>
  <widget class="QLabel" name="friendAvatar">
   <property name="geometry">
    <rect>
     <x>412</x>
     <y>102</y>
     <width>158</width>
     <height>158</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>friendAvatar</string>
   </property>
  </widget>
  <widget class="QFrame" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>570</width>
     <height>493</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/chatDlg_Bg.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="face">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>304</y>
     <width>25</width>
     <height>25</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/pixmap/image/faceButton.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="chatLogButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>285</y>
     <width>80</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>聊天记录</string>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="GifTextEdit" name="log">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>10</y>
     <width>412</width>
     <height>471</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'åææ¥·ä½'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="friend_sign">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>48</y>
     <width>400</width>
     <height>35</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>friendsign</string>
   </property>
  </widget>
  <zorder>bg</zorder>
  <zorder>avatar</zorder>
  <zorder>btn_min</zorder>
  <zorder>btn_close</zorder>
  <zorder>friend_name</zorder>
  <zorder>announcement</zorder>
  <zorder>messageEdit</zorder>
  <zorder>closeButton</zorder>
  <zorder>sendMessageButton</zorder>
  <zorder>saveFileButton</zorder>
  <zorder>refuseButton</zorder>
  <zorder>sendfileButton</zorder>
  <zorder>ownAvatar</zorder>
  <zorder>friendAvatar</zorder>
  <zorder>face</zorder>
  <zorder>fileTip</zorder>
  <zorder>chatLogButton</zorder>
  <zorder>log</zorder>
  <zorder>friend_sign</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GifTextEdit</class>
   <extends>QTextEdit</extends>
   <header>giftextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
